test/e2e-test/rule-test/config/testcases-hl7v2.js

```diff
@@ -5,9 +5,25 @@
 
 // design the test cases
 var ruleSets = require('./rule-sets'); 
-var hl7v2Testcases = [
-    { dataFile: 'ADT01-28.hl7', templateFile: 'ORU_R01.hbs', testRules: ruleSets.commonRules}
-];
+// var hl7v2Testcases = [
+//     { dataFile: 'ADT01-28.hl7', templateFile: 'ORU_R01.hbs', testRules: ruleSets.commonRules}
+// ];
+
+// module.exports = hl7v2Testcases;
 
-module.exports = hl7v2Testcases;
+const patterns = [
+    { template: 'ADT_A01.hbs', data: [ 'ADT01-23.hl7', 'ADT01-28.hl7' ] },
+    { template: 'VXU_V04.hbs', data: [ 'VXU.hl7' ] },
```

Comment:
We can add "IZ_1_..." here. It is a VXU_V04 message.

Reply:
Sure, and there is also a potential error. Since the line ending character in `IZ_1_1.1_Admin_Child_Max_Message` is CRLF before, and there are many `^M` substrings inside it, the file system will incorrectly take `^M` substrings as `CRLF` ending characters (since the `^M` is the visible display for ascii character `0xD`, or the `\r` character). It has been fixed yet.